packages/docs/src/data/api/rules/match-pattern.ts

```diff
@@ -0,0 +1,22 @@
+import { Api } from '~/types';
+import { ruleMessages } from '../shared/ruleMessages';
+
+export const api: Api = {
+	matchPattern: {
+		arguments: [
+			{
+				name: 'pattern',
+				type: 'RegExp',
+				description: 'Le motif à respecter.'
+			},
+			{
+				name: 'errorMessages',
+				type: 'ErrorMessages<number>',
+				description: 'Les messages d’erreur personnalisés.',
+			},
+			...ruleMessages(`{
+				default: 'Le format saisi ne correspond pas.',
+			}`)
```

Comment:
```suggestion
	default: 'Le format saisi ne correspond pas au format attendu.',
}`)
```
il faut absolument faire attention à l'indentation du code ici car vu qu'on utilise des templates literals, tous les caractères sont inclus dans la chaîne, du coup l'affichage n'est pas bon dans la documentation

![screenshot-647dc9ecb5874007a05dbf4a--digital-design-system netlify app-2023 06 07-18_09_55](https://github.com/assurance-maladie-digital/design-system/assets/10298932/60a12497-9203-4fa8-a2c4-d1dc689f9291)